### PR TITLE
Feature/core docs

### PR DESCRIPTION
**Pull request by emoralesb05**
## Description

Updated rest of core docs with ngModule.
https://github.com/Teradata/covalent/issues/65

### What's included?

- Updated `stepper` docs with ngModule and removed directives from examples.
- Updated `expansion` docs and removed directives from examples.
- Updated `media` docs and removed directives/providers from examples.

#### Test Steps

- [x] `ng serve`
- [x] Go to docs.
- [x] :tada: 

##### Screenshots or link to CodePen/Plunker/JSfiddle
image

### CODE REVIEW

File: src/app/components/components/expansion-panel/expansion-panel.component.html [hunk 1]
```diff
@@ -178,6 +176,21 @@ <h3 class="md-subhead">md-padding class</h3>
         }
       ]]>
     </td-highlight>
+    <h3>Setup:</h3>
+    <p>Import the [CovalentCoreModule] using the <code>forRoot()</code> method in your NgModule:</p>
+    <td-highlight lang="typescript">
+      <![CDATA[
+        import { CovalentCoreModule } from '@angular/core';
```
Comment (ilsiepotamus) [inline src/app/components/components/expansion-panel/expansion-panel.component.html]: should this be `@covalent/core`?

File: src/app/components/components/media/media.component.html [hunk 2]
```diff
@@ -88,6 +87,21 @@ <h3 md-line> {{attr.name}}: <span>{{attr.type}}</span></h3>
     </td-highlight>
     <p>Note: Always unsubscribe from [Observable] objects when not using them anymore.</p>
     <p>A good way of doing it is in the <code>ngOnDestroy</code> component life-cycle hook provided by the [OnDestroy] interface.</p>
+    <h3>Setup:</h3>
+    <p>Import the [CovalentCoreModule] using the <code>forRoot()</code> method in your NgModule:</p>
+    <td-highlight lang="typescript">
+      <![CDATA[
+        import { CovalentCoreModule } from '@angular/core';
```
Comment (ilsiepotamus) [inline src/app/components/components/media/media.component.html]: same as above

File: src/app/components/components/steps/steps.component.html [hunk 3]
```diff
@@ -78,6 +77,21 @@ <h3 md-line> {{attr.name}}: <span>{{attr.type}}</span></h3>
         }
       ]]>
     </td-highlight>
+    <h3>Setup:</h3>
+    <p>Import the [CovalentCoreModule] using the <code>forRoot()</code> method in your NgModule:</p>
+    <td-highlight lang="typescript">
+      <![CDATA[
+        import { CovalentCoreModule } from '@angular/core';
```
Comment (ilsiepotamus) [inline src/app/components/components/steps/steps.component.html]: same

File: src/app/components/components/expansion-panel/expansion-panel.component.html (hunk 1 above)
Comment (emoralesb05) [inline src/app/components/components/expansion-panel/expansion-panel.component.html]: dammit, copy paste is my worse enemy haha

**Comment by kyleledbetter** [comment]
is TdMediaToggleDirective still a directive that's usable? can you add docs to use it in here?

**Comment by emoralesb05** [comment]
Its there in the docs https://github.com/Teradata/covalent/blob/f96b8d6f901e1e9cecf91a4a0831e2a05d040e3b/src/app/components/components/media/media.component.html#L108

**Pull request merged**